webapp/components/generic/SearchableInput/SearchableInput.vue

```diff
@@ -170,58 +149,32 @@ export default {
   },
 }
 </script>
+
 <style lang="scss">
 .searchable-input {
+  position: relative;
   display: flex;
-  align-self: center;
+  align-items: center;
   width: 100%;
-  position: relative;
-  $padding-left: $space-x-small;
-  .option-with-heading {
-    margin-top: $space-x-small;
-    padding-top: $space-xx-small;
-  }
-  .flex-span {
-    display: flex;
-    flex-wrap: wrap;
+
+  .ds-form-item {
+    flex-basis: 100%;
+    margin-bottom: 0;
   }
+
   .ds-select-dropdown {
-    transition: box-shadow 100ms;
     max-height: 70vh;
+    box-shadow: $box-shadow-x-large;
   }
-  &.is-open {
-    .ds-select-dropdown {
-      box-shadow: $box-shadow-x-large;
-    }
-  }
-  .ds-select-dropdown-message {
-    opacity: 0.5;
-    padding-left: $padding-left;
+
+  .option-with-heading {
+    margin-top: $space-x-small;
+    padding-top: $space-xx-small;
   }
-  .search-clear-btn {
-    right: 0;
-    z-index: 10;
+
+  .base-button {
     position: absolute;
-    height: 100%;
-    width: 36px;
-    cursor: pointer;
-  }
-  .ds-select {
-    z-index: $z-index-dropdown + 1;
-  }
-  .ds-select-option-hover {
-    .ds-text-size-small,
-    .ds-text-size-small-x {
-      color: $text-color-soft;
-    }
-  }
-  .field {
-    width: 100%;
-    display: flex;
-    align-items: center;
-  }
-  .control {
-    width: 100%;
+    right: $space-xx-small;
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-15T16:32:11Z" title="Wednesday, January 15th 2020, 5:32:11 pm +01:00">Jan 15, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -170,58 +149,32 @@ export default {
   },
 }
 </script>
+
 <style lang="scss">
 .searchable-input {
+  position: relative;
   display: flex;
-  align-self: center;
+  align-items: center;
   width: 100%;
-  position: relative;
-  $padding-left: $space-x-small;
-  .option-with-heading {
-    margin-top: $space-x-small;
-    padding-top: $space-xx-small;
-  }
-  .flex-span {
-    display: flex;
-    flex-wrap: wrap;
+
+  .ds-form-item {
+    flex-basis: 100%;
+    margin-bottom: 0;
   }
+
   .ds-select-dropdown {
-    transition: box-shadow 100ms;
     max-height: 70vh;
+    box-shadow: $box-shadow-x-large;
   }
-  &.is-open {
-    .ds-select-dropdown {
-      box-shadow: $box-shadow-x-large;
-    }
-  }
-  .ds-select-dropdown-message {
-    opacity: 0.5;
-    padding-left: $padding-left;
+
+  .option-with-heading {
+    margin-top: $space-x-small;
+    padding-top: $space-xx-small;
   }
-  .search-clear-btn {
-    right: 0;
-    z-index: 10;
+
+  .base-button {
     position: absolute;
-    height: 100%;
-    width: 36px;
-    cursor: pointer;
-  }
-  .ds-select {
-    z-index: $z-index-dropdown + 1;
-  }
-  .ds-select-option-hover {
-    .ds-text-size-small,
-    .ds-text-size-small-x {
-      color: $text-color-soft;
-    }
-  }
-  .field {
-    width: 100%;
-    display: flex;
-    align-items: center;
-  }
-  .control {
-    width: 100%;
+    right: $space-xx-small;
```


`CSS` magic! down from `52` to `25` lines!!! impressive!